brave-lists/debounce.json

```diff
@@ -170,7 +170,8 @@
   },
   {
     "include": [
-      "*://*.getprice.com.au/prodhits.aspx?*"
+      "*://*.getprice.com.au/prodhits.aspx?*",
+      "*://search.app/?*"
```

Comment:
Can you please insert it at the top of the list, just before the other entry so that we don't have to touch the existing entry (and add noise to the `git blame`)?

Reply:
Okay! Done.